pyworkflow/mapper/xmlmapper.py

```diff
@@ -26,11 +26,7 @@
 # **************************************************************************
 
 from os.path import exists
-
-try:
```

Comment:
I think we can get rid of xmlmapper completely?

Reply:
sure, but that was not my goal in this PR :)